src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -217,6 +217,8 @@ internal string GetDebuggerDisplay()
                                 }
                                 else
                                 {
+                                    Binder.ReportDiagnosticsIfObsoleteInternal(diagnostics, importedType, usingDirective.Name, importedType.ContainingSymbol, BinderFlags.None);
```

Comment:
Should the error be report on the `using static` statement or only on the use of that static type? For aliases such as `using A = N.C;`, we only report errors on the use of `A` whe `C` is marked `[Obsolete]`.